RELEASE-NOTES.txt

```diff
@@ -8,6 +8,7 @@
 * [*] Login Epilogue: Fixed bug where if no sites available, then new login epilogue screen without clickable sites and with "create new site" is shown instead of no sites empty "My Site" view. [https://github.com/wordpress-mobile/WordPress-Android/pull/15944]
 * [*] Fixes flickering when changing the preview mode in Page or Site design picker [https://github.com/wordpress-mobile/WordPress-Android/pull/15943]
 * [*] Reader: Admins and Editors can edit comments from Reader [https://github.com/wordpress-mobile/WordPress-Android/pull/15957]
+* [*] Block Editor: Stop editor from sometimes closing after composing a story block [https://github.com/wordpress-mobile/WordPress-Android/pull/15960]
```

Comment:
I wasn't exactly sure how best to phrase this release note since the crash wouldn't really look like a crash to the user, and it would likely only occur sporadically when memory was especially low.

Reply:
Are you saying this because if the editor crashed in the JS runtime then the user would remain in the enclosing Native app/runtime?

Perhaps don't phrase the fix from the perspective of the symptom, but rather the problem, in the release notes?

Reply:
I read the changes before I read the description. :)

Reply:
> Perhaps don't phrase the fix from the perspective of the symptom, but rather the problem, in the release notes?

The problem (improperly recreating an object when the editor activity gets destroyed by the os) seems pretty technical, and I'm having trouble thinking of a way to phrase that in a way that would make sense for user-facing release notes. I was focusing on the symptom because that is what the user would actually have experienced.

Maybe I'm just not seeing it, though. Could you give me an example of how you might phrase the release note?

Reply:
That does make sense. I feel like it makes more sense to focus on the editor closing/crashing compared to the fact that the story block isn't updated if/when the user decides to reopen the editor. 

I updated the wording a bit, and although I still don't love it, I think it reads better now:

```
Block Editor: Stop editor from unexpectedly closing while composing a story block when memory is low
```

wdyt?